src/GSMSSLClient.cpp

```diff
@@ -17,8 +17,19 @@
   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
+#include "utility/GSMRootCerts.h"
```

Comment:
This adds all the certificates in SAMD flash every time GSMSSLClient is not being stripped by the linker? If the root certificates load is ram-only it makes sense, otherwise I'd go for a "provisioning" sketch

Reply:
I think we are ok, I compiled the `GsmWebClient` example from master the size was `35272 ` bytes, with this branch it was `35640`.

The `GsmSSLWebClient` was `35512` bytes in master and  `49024` bytes on the branch.

Reply:
:+1: